downstream/modules/hub/con-offline-token-active.adoc

```diff
@@ -3,16 +3,19 @@
 
 = Keeping your offline token active
 
-Keeping an offline token active is useful when an application needs to perform an action on behalf of the user, like a routine data backup, even when the user is offline.
-
 Offline tokens expire after 30 days of inactivity. You can keep your offline token from expiring by periodically refreshing your offline token.
 
+Keeping an offline token active is useful when an application performs an action on behalf of the user; when the user is offline, such as a routine data backup.
```

Comment:
I'm not sure I agree with this update. A semicolon should connect two complete clauses (i.e. they should be able to stand on their own); the second clause isn't complete. To keep the semicolon I would change to: 

Keeping an online token active is useful when an application performs an action on behalf of the user; for example, this allows the application to perform a routine data backup when the user is offline.

Reply:
Your suggestion is much better. This is fixed.